backend/kernelCI_app/views/hardwareDetailsView.py

```diff
@@ -190,8 +203,24 @@ def pass_in_filters(self, data, filters):
 
         return True
 
+    def should_jump_test(
+        self,
+        table_test: Literal["boot", "test"],
+        record: Dict,
+        filters_map: Dict[str, List[Dict]],
+        processed_test: Set[str],
+    ) -> bool:
+        has_test_been_processed = record["id"] in processed_test
+        if has_test_been_processed:
+            return True
```

Comment:
nit: I don't think there is a need for this one-use variable, you could make the if statement directly